project.clj

```diff
@@ -4,11 +4,12 @@
   :license {:name "GNU General Public License v3.0"
             :url "https://www.gnu.org/licenses/gpl-3.0.en.html"}
   :dependencies [[org.clojure/clojure "1.11.3"]]
-  :source-paths ["src/clj" "examples"]
-  :java-source-paths ["src/java"]
-  :resource-paths ["resources"]
-
-  :profiles {:examples {:dependencies [[quil "4.3.1563"]]
-                        :source-paths ["src/clj" "examples"]}}
+  :source-paths ["src" "examples"]
+  :java-source-paths ["src"]
+  :profiles {:examples {:dependencies [[quil/quil "4.3.1563"]]
+                        :source-paths ["src" "examples"]}
+             :cljs {:source-paths ["src" "test"]
+                    :dependencies [#_[org.clojure/clojurescript "1.11.123"] ;; TODO: Make cljs dep explicit
```

Comment:
what needs to be done for this?

Reply:
Somehow, when i tried to use an explicit cljs version i got this:
```IllegalArgumentException: No matching method legacySetOutputFeatureSet found taking 1 args for class com.google.javascript.jscomp.CompilerOptions```

But i found a workaround in [shadow-cljs project.clj](https://github.com/thheller/shadow-cljs/blob/master/project.clj)

And now it works. So i'll commit this as well :-)